src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/DataConvert/DataConvertEngineTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Azure.ContainerRegistry;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Features.Operations.DataConvert;
+using Microsoft.Health.Fhir.Core.Features.Operations.DataConvert.Models;
+using Microsoft.Health.Fhir.Core.Messages.DataConvert;
+using Microsoft.Health.Fhir.TemplateManagement;
+using Microsoft.Health.Fhir.TemplateManagement.Models;
+using NSubstitute;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Shared.Core.UnitTests.Features.Operations.DataConvert
+{
+    public class DataConvertEngineTests
+    {
+        private const string DefaultTemplateReference = "microsofthealth/fhirconverter:default";
```

Comment:
>DefaultTemplateReference [](start = 29, length = 24)

I think I saw that string before...
Can `DefaultTemplateReference` from DataConvertEngine become internal so it can be reference in this class instead of defining it second time?

Reply:
Sure. Have made it a reference from the template management library